eng/pipelines/templates/jobs/smoke.tests.yml

```diff
@@ -136,6 +141,9 @@ jobs:
           $packageOverrides = @()
           $packageOverrides += (dir env: | Where-Object { $_.Name.StartsWith("SMOKE_PACKAGE_") }).Value
 
+          $packageArtifact = Get-ChildItem $(Pipeline.Workspace)/${{parameters.ArtifactName}}/${{parameters.Artifact.name}}/*.tgz
```

Comment:
Similar to python lets make sure we actually find a package that we expected to find.